composer.json

```diff
@@ -43,6 +45,9 @@
     "typo3/cms": {
       "cms-package-dir": "{$vendor-dir}/typo3/cms",
       "web-dir": ".Build/Web"
+    },
+    "helhum/typo3-console": {
+      "install-extension-dummy": false
```

Comment:
is this required?

Reply:
Yeah, I'm using the typo3_console to generate the PackageStates.php for testing purpose, without that I get that the packages states is not present or invalid. 

The helhum/typo3-console in composer.json is to avoid a warning, when running the composer update directly in extension folder.